spec/paperclip/storage/s3_spec.rb

```diff
@@ -342,18 +350,19 @@ def counter
 
   context "An attachment that uses S3 for storage and has spaces in file name" do
     before do
-      rebuild_model styles: { large: ['500x500#', :jpg] },
+      rebuild_model(
+        styles: { large: ['500x500#', :jpg] },
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.